src/mmw/js/src/modeling/views.js

```diff
@@ -494,19 +502,37 @@ var ModelingResultsWindow = Marionette.LayoutView.extend({
     animateIn: function() {
         var self = this;
 
-        this.$el.animate({ height: '55%' }, 400, function() {
+        this.$el.animate({ height: '55%', 'min-height': '300px' }, 200, function() {
```

Comment:
Setting both `animateIn` and `animateOut` to 200 gives a more consistent feel than the previous 400 for in and 100 for out.